src/librbd/ImageCtx.h

```diff
@@ -26,6 +26,7 @@
 
 #include "cls/rbd/cls_rbd_client.h"
 #include "librbd/LibrbdWriteback.h"
+#include "librbd/ObjectMap.h"
```

Comment:
Minor: the downside to making ObjectMap a non-pointer is the header dependencies to ObjectMap now basically expand to every source file in librbd.